src/include/mom_func.h

```diff
@@ -143,6 +143,8 @@ enum hup_action {
  */
 extern enum hup_action	call_hup;
 
+extern int mock_run;
```

Comment:
I suggest (if possible) then moving this in "mock_run.h"

Reply:
it's consistent with the way other globals, especially the ones set in getopt, are declared in mom right?

Reply:
ok, I didn't know that, but in my view, we should keep all the required things in one place for a particular interface... but up you, if you want to keep here, I am fine with it.